src/components/NavigationAccount.vue

```diff
@@ -129,16 +129,32 @@ export default {
 		},
 		deleteAccount() {
 			const id = this.account.id
+			OC.dialogs.confirmDestructive(
+				t('mail', 'Fully delete {accountId}\'s account', { accountId: this.account }),
+				t('mail', 'Account deletion'),
+				{
+					type: OC.dialogs.YES_NO_BUTTONS,
+					confirm: t('mail', 'Delete {accountId}\'s account', { accountId: this.account }),
```

Comment:
This doesn’t seem to work as it shows `Delete {accountId}'s account` on the button instead of the actual email address.

Reply:
Yes, i am aware of that. Thats why i left it in draft because its not ready for review yet :) but thank you for the suggestions will replace them asap

Reply:
Thought so, yes :)

Reply:
Just tested it and this still happens, am I missing something?